workflow/envs/filter_reads.yaml

```diff
@@ -2,7 +2,8 @@ channels:
   - conda-forge
 dependencies:
   - rust-script >=0.17.0
-  - rust >=1.51
+  - rust >=1.58
+  - cryptography >=36.0
   - c-compiler =1.3
   - cmake =3.21 
   - make =4.3
```

Comment:
Can these packages now be removed?

Reply:
You are right. Just did a successful rerun without them.